recipe/bld.bat

```diff
@@ -5,26 +5,6 @@ copy %RECIPE_DIR%\missing_files\*.c tests\
 
 set PATH=%PREFIX%\cmake-bin\bin;%PATH%
 
-mkdir build_static && cd build_static
-
-:: Build static libraries
-cmake -GNinja ^
-    -D CMAKE_BUILD_TYPE=Release ^
-    -D BUILD_SHARED_LIBS=OFF ^
-    -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
-    -D CMAKE_PREFIX_PATH=%LIBRARY_PREFIX% ^
-    -D ENABLE_ZLIB_COMPRESSION=ON ^
-    -D BUILD_EXAMPLES=OFF ^
-    -D BUILD_TESTING=OFF ^
-	%SRC_DIR%
-IF %ERRORLEVEL% NEQ 0 exit 1
-
-ninja
-IF %ERRORLEVEL% NEQ 0 exit 1
-ninja install
-IF %ERRORLEVEL% NEQ 0 exit 1
-
-cd ..
```

Comment:
Why is the static build being removed?

Reply:
I added it accidentially to the initial 1.9.0 build as the existence test for `libssh2.lib` was failing. BUT I was tricked into that by the bug this PR is fixing. We need `libssh2.lib` to be the import library for `libssh2.dll`, not the static library.

In general, we shouldn't be building static libraries at all in the main package as our update/security machinery works only with dynamic linkage. One day in the coming months, I'll write a CFEP with reasoning/details on why we shouldn't do this.